src/components/views/ScreeningsList.vue

```diff
@@ -0,0 +1,54 @@
+<script>
+import { defineComponent } from 'vue';
+import ScreeningsCard from '@/components/views/ScreeningsCard.vue';
+
+export default defineComponent({
+  components: { ScreeningsCard },
+  props: {
+    movies: {
+      type: Array,
+      required: true,
+    },
+    daySelected: {
+      type: Date,
+      default: new Date().toISOString().slice(0, 10)
+    },
+    movieSelected: {
+      type: String,
+      default: 'All movies',
+    }
+  },
+  data() {
+    return {
+      screenings: []
+    }
+  },
+  computed: {
+    filteredMovies() {
+      if (this.movieSelected === 'All movies') {
+        return this.movies
+      } else {
+        return this.movies.filter(movie => this.movieSelected === movie.title)
+      }
```

Comment:
Also we shouldn't check just by title! We should check by `id` field!
The title of the movies **can** be duplicated - hence when people talk about movies we usually type it in a `Name (Year)` fashion - i.e. Dune (1984) vs Dune (2021) even though the name is still **Dune** 😄

Reply:
Good point indeed! 💪